pom.xml

```diff
@@ -96,7 +96,7 @@
         <maven.plugin.build.helper.version>3.2.0</maven.plugin.build.helper.version>
         <maven.plugin.compiler.version>3.8.1</maven.plugin.compiler.version>
         <!-- DO NOT bump 4.4.0, see https://github.com/NetEase/kyuubi/pull/441 -->
```

Comment:
Have a verification that #441 still exists in 4.5.3, I think we need to figure out what happens in scala maven plugin before doing this upgrading.

Reply:
Now I will upgrade the version of `scala-maven-plugin` to `4.5.3` and pass the local verification, and run the `build/dist --tgz` command to package normally

Reply:
Yes, it also passed ci but not the case reported by #441

Reply:
What should I do?

Reply:
We need to figure out what happens in scala maven plugin cause #441 first. 
If it's an accident change, fix it in scala maven plugin;
if scala maven plugin team confirme the change is by design and they don't think #441 should be supported, we can do upgrade and ignore #441.

Reply:
Let's add a new comment here and skip this version too.

```
<!-- DO NOT bump 4.4.0, see https://github.com/NetEase/kyuubi/pull/441 -->
<!-- DO NOT bump 4.5.3, see https://github.com/NetEase/kyuubi/issues/708 -->
```